src/library_syscall.js

```diff
@@ -660,7 +666,9 @@ var SyscallsLibrary = {
   __syscall140: function(which, varargs) { // llseek
     var stream = SYSCALLS.getStreamFromFD(), offset_high = SYSCALLS.get(), offset_low = SYSCALLS.get(), result = SYSCALLS.get(), whence = SYSCALLS.get();
     var offset = offset_low;
+#if WASM_BACKEND == 0
```

Comment:
Right, the high bits are assumed to be unset and unused, hence the assert. I'll need to investigate more I think and figure out why wasm_backend has a value here, as opposed to just writing it off as undefined behavior misbehaving.

It might be equally sane to explicitly pass `0` for the high bits `#ifdef __EMSCRIPTEN__` in the C source.